src/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -73,8 +78,8 @@
             "      R=reference_sequence.fasta " +
             "</pre>" +
             "Please see " +
-            "<a href='https://broadinstitute.github.io/picard/picard-metric-definitions.html#CollectWgsMetrics.WgsMetrics'>" +
-            "the WgsMetrics documentation</a>for detailed explanations of the output metrics." +
+            "<a href='https://broadinstitute.github.io/picard/picard-metric-definitions.html#CollectWgsMetrics.CollectWgsMetrics'>" +
```

Comment:
Still a broken link. This should end in CollectWgsMetrics.WgsMetrics as it needs to point to the metric not the collector. also please revert the text in the `<a>`. 

while you're here, could you please add a space after the `</a>`?